enterprise-suite/gotests/util/prometheus/prometheus.go

```diff
@@ -32,8 +33,14 @@ type Connection struct {
 
 func (p *Connection) Query(query string) (*PromResponse, error) {
 	addr := fmt.Sprintf("%v/api/v1/query?query=%v", p.url, url.QueryEscape(query))
-
-	resp, err := http.Get(addr)
+	// Some of the tests with openshift clusters timedout in 20 seconds, so adding a timeout for
+	// 45 seconds. TODO: Remove this time out once sub-queries are available.
```

Comment:
This comment isn't correct. Can you remove it or change it? We're already using subqueries.